### ISSUE TITLE
Emulator v3.5.36 not connecting to Bot

### ISSUE THREAD

**Issue by nitinkallikat** [Microsoft/BotFramework-Emulator#481]
My bot framework emulator auto-updated to v3.5.36 (from v3.5.35) yesterday, and has stopped connecting to my Bot.  It doesn't give any feedback or error. I tried installing the earlier version, which made it work again, but then the emulator auto updates itself on next restart, and the issue appears again.

I am using the BotBuilder v3.11.0 for the Bot 

![image](https://user-images.githubusercontent.com/14120921/39713873-24b17e7e-51dd-11e8-99ff-66f864860fc1.png)

**Comment by eanders-MS** [comment]
Could you send a fiddler trace? I wonder if the emulator is attempting to send anything at all to your bot. In the meantime to unblock you, you can downgrade to the previous version of the emulator, but you must block it from auto updating. You can do this by creating a shortcut to the installed executable and adding the command line argument `--no-update`. But there's a complication with this. You must add a placeholder argument in front of it, so `--foo --no-update`

**Comment by nitinkallikat** [comment]
Thanks, I was able to figure out the issue after checking the Fiddler trace as you requested. The problem was that I was using a certificate issued to our remote test host for SSL connection on my localhost, and the certificate validation was failing due to host name mismatch. I went around it by using the test host URL in the emulator and adding it to my hosts file

It seems earlier versions of the emulator handled this internally somehow, but now it doesn't?
